### ISSUE TITLE
[Bug]: No error message when using multiple Ledger devices

### ISSUE THREAD

**Issue by T-Low2142** [iotaledger/firefly#2270]
### Issue description

1. Missing Error message when you try to send an amount of IOTA with the confirmation of a wrong ledger Device.
2. Wrong behaviour after this failure on the firefly account.

### Expected behaviour

There should applie an error message like "the confirmation device is wrong".

### Actual behaviour

First Error:
I have two Ledger Devices. NanoS and NanoX. 
I created on each Ledger Device an Account. Then I sent my IOTA from the NanoS account to the NanoX account. 
But I confirmed the transaction with my NanoS device. There appeared no error message or information. Just appeared something like "successfull". After some time I could see the following error via the transaction in the explorer:
6111d0436e905eb55a963714d269e8709b345f175de29a99b76e5bf40fbc8f2d

Second Error: The Firefly Account show that I have my balance on the Nano S account but do net let me move the balance again. It shows me the error message that I do not have enough coins to send. So your discord support (tOOk) helped me to inform me I have to make a new account on this device. I did and it worked correctly. So I had again my coins on the nano S and could no correctly send them to the nanoX.

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

"Actual Behaviour"

### Diagnostics

```shell
Its a mess that I cannot copy paste.... 

v1.3.3
plattformversion 10.0.22000
Architecture x64
```

### Errors

```shell
"insufficientFunds: insufficient funds 0/1398100000 available or input address used as output
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by begonaalvarezd** [comment]
Hi, is this still an issue in the latest version?

**Comment by begonaalvarezd** [comment]
The network was recently upgraded and we released a new version firefly iota 2.0.2. You need iota ledger app 0.8.6, if you cant see it in the ledger store need to upgrade your firmware

**Comment by begonaalvarezd** [comment]
Closing issue due to inactivity, please update to the latest version and reopen the issue if the problem persist 🌷
